Jenkinsfile.cd

```diff
@@ -636,6 +761,9 @@ def publishingRCtoStable() {
                 echo 'Moving Ubuntu RC artifacts to Stable: libnullpay'
                 publishLibnullpayDebRCtoStable(testEnv)
 
+                echo 'Moving Android RC artifacts to Stable: libindy'
+                androidPublishing("1")
```

Comment:
this call will results in full rebuild android artifacts from scratch. If I understand build script correctly, we can just use RC archive, rename it and publish to stable directory

Reply:
Fixed!